benches/benches/wgpu-benchmark/shader.rs

```diff
@@ -420,28 +450,30 @@ fn backends(c: &mut Criterion) {
                 zero_initialize_workgroup_memory: true,
             };
             for input in &inputs.inner {
-                let module = input.module.as_ref().unwrap();
-                let info = input.module_info.as_ref().unwrap();
-                for ep in module.entry_points.iter() {
-                    let pipeline_options = naga::back::glsl::PipelineOptions {
-                        shader_stage: ep.stage,
-                        entry_point: ep.name.clone(),
-                        multiview: None,
-                    };
-
-                    // might be `Err` if missing features
-                    if let Ok(mut writer) = naga::back::glsl::Writer::new(
-                        &mut string,
-                        module,
-                        info,
-                        &options,
-                        &pipeline_options,
-                        naga::proc::BoundsCheckPolicies::default(),
-                    ) {
-                        let _ = writer.write(); // might be `Err` if unsupported
+                if input.options.targets.unwrap().contains(Targets::GLSL) {
```

Comment:
Nit: lets make these early outs - so `if !condition { continue; }` to reduce rightward creep